services/figgy.md

```diff
@@ -0,0 +1,79 @@
+# Figgy
+
+## Ingest from Isilon
+
+Each of Figgy's servers has SMB mounts to a variety of places on the Isilon, all
+under `/mnt/hydra_sources`. They are the following:
+
+1. archives
+    * Contains source TIFFs for digitized finding aids.
+1. ingest_scratch
+    * Working directory for Figgy developers to move content to in order to bulk
+      ingest.
+1. maplab
+1. pudl
+    * Contains source TIFFs for content found in PUDL. Primarily used for
+      migrating to Figgy.
+1. studio
+    * Old studio staging server - not currently used for ingest.
+1. studio_new
+    * New studio staging server - the digitization studio saves images here for
+      ingest into Figgy.
+
+If you need to ingest content, the first step is to make sure the data is in one
+of these shares. If it's new content, rather than being migrated, it should
+ideally be in `ingest_scratch`. If you try to ingest from a location not in
+these shares, background jobs will run on other servers which will be unable to
+access the files. Currently content can be ingested in three
+forms:
+
+### Ingesting METS
+
+Items coming from the PUDL consists of a METS file as well as a series of
+images. The METS files have links to the source TIFFs and the mounts should be
+set up appropriately. To ingest a METS file do the following:
+
+1. `ssh deploy@lib-proc6`
+2. `cd /opt/figgy/current`
+3. `RAILS_ENV=production FILE=/mnt/hydra_sources/ingest_scratch/x/my_file.mets bundle exec rake import:mets`
+
+### Ingesting Bags
```

Comment:
We should clarify because there will be bags that we'll be ingesting, the a/v bags. Should we call this "Ingesting Figgy Preservation Bags"?